libimage/runtime.go

```diff
@@ -389,16 +389,17 @@ func (r *Runtime) lookupImageInDigestsAndRepoTags(name string, options *LookupIm
 		return nil, "", err
 	}
 
-	if !shortnames.IsShortName(name) {
-		named, err := reference.ParseNormalizedNamed(name)
-		if err != nil {
-			return nil, "", err
-		}
-		digested, hasDigest := named.(reference.Digested)
-		if !hasDigest {
-			return nil, "", errors.Wrap(storage.ErrImageUnknown, name)
-		}
+	ref, err := reference.Parse(name)
```

Comment:
Absolutely non-blocking:

```suggestion
	ref, err := reference.Parse(name) // Warning! This is not ParseNormalizedNamed
```
or possibly even using `unnormalizedRef`/`unnormalizedNamed` variable names.